include/real/real_math.hpp

```diff
@@ -0,0 +1,298 @@
+#ifndef BOOST_REAL_MATH_HPP
+#define BOOST_REAL_MATH_HPP
+
+#include <tuple>
+#include "real/exact_number.hpp"
+#include "real/real_exception.hpp"

```

Comment:
Please use <> includes for library includes.

Reply:
Done that. Next commit will have these changes.